contracts/tests/recipientRegistry.ts

```diff
@@ -827,16 +828,14 @@ describe('Optimistic recipient registry', () => {
       }
     })
 
-    it('allows anyone to submit removal request', async () => {
+    it('allows owner to submit removal request', async () => {
       await registry.addRecipient(recipientAddress, metadata, {
         value: baseDeposit,
       })
       await provider.send('evm_increaseTime', [86400])
       await registry.executeRequest(recipientId)
 
-      const requestSubmitted = await registry
-        .connect(requester)
-        .removeRecipient(recipientId, { value: baseDeposit })
+      const requestSubmitted = await registry.removeRecipient(recipientId)
```

Comment:
I've removed `baseDeposit` from every `removeRecipient` call since we don't require it anymore (since it is onlyOwner).